aws_lambda_powertools/logging/logger.py

```diff
@@ -387,15 +387,28 @@ def structure_logs(self, append: bool = False, **keys):
             formatter = self.logger_formatter or LambdaPowertoolsFormatter(**log_keys)  # type: ignore
             self.registered_handler.setFormatter(formatter)
 
-    def set_correlation_id(self, value: str):
+    def set_correlation_id(self, value: Optional[str]):
         """Sets the correlation_id in the logging json
 
         Parameters
         ----------
-        value : str
+        value : str, optional
+            Value for the correlation id. None will remove the correlation_id
+        """
+        if value is None:
+            self.remove_keys(["correlation_id"])
+        else:
+            self.append_keys(correlation_id=value)
```

Comment:
Keys with `None` value get removed by default when emitting a log message, so this wouldn't be necessary: https://github.com/awslabs/aws-lambda-powertools-python/blob/89e8151b5c930ba79d685d05a83ae362cf94839d/aws_lambda_powertools/logging/formatter.py#L126

Reply:
resolved.